docs/use_cases/model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratosphereQBO.py

```diff
@@ -0,0 +1,158 @@
+"""
+QBO Phase plots and QBO Index: UserScript, Stat-Analysis
+==========================================================================
+
+model_applications/
+s2s_stratosphere/
+UserScript_fcstGFS_obsERA_StratosphereQBO.py
+
+"""
+
+##############################################################################
+# Scientific Objective
```

Comment:
The scientific objective section should answer the question, "why was this use case created?". Right now, this write-up tells a lot about "what" is created. Very important, but better suited for the Expected Output and METplus Wrokflow sections. 
Try to focus on why this use case is necessary and important to include in the repository.

Reply:
I actually have a number of these to fix.  Since they are outside my area of expertise they need interaction with SMEs.  Due to time constraints, the plan is to leave this section as is, and it will be fixed along with a few other use cases where we need to edit this section.

Reply:
That's a good reason to keep moving along with this as is, then. When this PR is successfully completed, it might be useful to create a new Github issue detailing the need to update the scientific objective of this use case so we don't forget about it. That would also help in case someone other than yourself has to track down the information.

Reply:
I have an issue created already so it's not forgotten.  I did up date everything that I know about QBO, which isn't much in how this use case is use scientifically for QBO